components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/import-file-dialog.js

```diff
@@ -16,266 +16,193 @@
+                        'source_view.container');
+                },
+
+                render: function () {
+                    var self = this;
+                    var app = this.app;
+                    var fileName;
+                    var notification_container = this.notification_container;
+                    var ImportLink = document.createElement("input");
+                    ImportLink.type = "file";
+                    ImportLink.name = "File";
+                    ImportLink.accept = ".siddhi";
+                    ImportLink.click();
+                    ImportLink.onchange = handleFiles;
+                    ImportLink.onclick = destroyClickedElement;
+                    ImportLink.style.display = "none";
+                    document.body.appendChild(ImportLink);
+
+                    function destroyClickedElement(event) {
+                        document.body.removeChild(event.target);
                     }
-                    return $(
-                        "<div style='z-index: 9999;' style='line-height: 20%;' class='alert alert-danger'" +
-                        " id='error-alert'>" + "<span class='notification'>" +
-                        errorMsg +
-                        "</span>" +
-                        "</div>");
-                }
 
-                var openConfigModal = fileImport.filter("#openConfigModal");
-                var openFileWizardError = fileImport.find("#openFileWizardError");
-                var location = fileImport.find("input").filter("#location");
-                var selectedFilePath = "";
+                    function handleFiles(event) {
+                        var files = event.target.files;
+                        var file = files[0];
+                        var reader = new FileReader();
 
-                var treeContainer  = fileImport.find("div").filter("#fileTree")
-                fileBrowser = new FileBrowser({container: treeContainer, application:app, fetchFiles:true,
-                    showWorkspace:false});
+                        fileName = file.name;
 
-                fileBrowser.render();
-                this._fileBrowser = fileBrowser;
+                        var existsResponse = existFileInPath({
+                            configName: fileName
+                        });
 
-                //Gets the selected location from tree and sets the value as location
-                this.listenTo(fileBrowser, 'selected', function (selectedLocation) {
-                    selectedFilePath = selectedLocation;
-                    var pathAttributes = selectedLocation.split(self.pathSeparator);
-                    var fileName = _.last(pathAttributes);
+                        if (existsResponse.error === undefined) {
+                            if (existsResponse.exists) {
+                                alertError("A file already exist in workspace with selected name.");
```

Comment:
Shall we add the file name to the error message

Reply:
Fixed with #1690